### ISSUE TITLE
Dates in Datepicker are not navigable using keyboard in IE

### ISSUE THREAD

**Issue by DhirajKhot** [OfficeDev/office-ui-fabric-react#10488]
### Environment Information

- **Package version(s)**: Latest
- **Browser and OS versions**: IE 11

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:
After opening the calendar, w can use the Tab button to navigate to the dates section.
Once a date is in focus, we should be able to navigate through the dates using arrow keys.
Black box border is shown for the date which is currently focused. This is current behavior with Chrome Browser. 
But for IE the black box border is not shown.

Refer the images below: 
Chrome browser - 
![image](https://user-images.githubusercontent.com/20378231/65139044-99c74b00-da29-11e9-929c-ebdea86189ab.png)

IE browser - 
![image](https://user-images.githubusercontent.com/20378231/65139148-c67b6280-da29-11e9-8a70-d215d6291f3c.png)

**Note -** There is no black border for date 20 September even if I have navigated to 20 date.

#### Actual behavior:

Border should be visible in all browsers including IE

#### Expected behavior:

IE browser does not show the border

**Comment by aneeshack4** [comment]
@DhirajKhot Are you sure you're using the latest version of IE11 and Fabric? This doesn't repro for me - the black box does get drawn during keyboarding: 
![image](https://user-images.githubusercontent.com/4161791/65183646-beff8d80-da18-11e9-8130-e04fe33d8fa0.png)
![image](https://user-images.githubusercontent.com/4161791/65183678-cd4da980-da18-11e9-84c8-f51b502f2bdd.png)

**Comment by DhirajKhot** [comment]
Hi @aneeshack4 

I am using the latest version of IE 11. 
![image](https://user-images.githubusercontent.com/20378231/65209099-ac3a9680-dab4-11e9-8250-bb3f08269afd.png)

I tried on multiple apps of office.com like To-Do, Planner, Project. 
I could repro the issue at all the places. 
I also tried to create a sample react app and just use the Calendar component of office fabric. 
I was able to repro the issue in IE 11.

**Comment by lorejoh12** [comment]
I am able to repro this with both the date-time package and the normal OUFR calendar in IE11. Will work on a fix.
